### PR TITLE
Make review.sh deal with git

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

File: resources.whatwg.org/build/review.sh [hunk 1]
```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
-set -o errexit
```
Comment (domenic) [inline resources.whatwg.org/build/review.sh]: Why remove this? If it's for diff, then I think a better solution would be to add `|| 0` to the end of that line

Comment (annevk) [reply resources.whatwg.org/build/review.sh]: When I try that on whatwg/html's equivalent I get `./review-draft.sh: line 26: 0: command not found`.

Comment (sideshowbarker) [reply resources.whatwg.org/build/review.sh]: I reckon Domenic meant `|| exit 0`

Comment (domenic) [reply resources.whatwg.org/build/review.sh]: It looks like `true` is what I meant. I will add suggested changes here.

File: resources.whatwg.org/build/review.sh [hunk 2]
```diff
@@ -15,4 +19,7 @@ sed 's/^Group: WHATWG$/&\
 '"Date: $(date +'%Y-%m-%d')/g" < "$INPUT_FILE" > "$REVIEW_DRAFT"
 echo "Created Review Draft at $REVIEW_DRAFT"
```
Comment (domenic) [inline resources.whatwg.org/build/review.sh]:
```suggestion
echo ""
echo "Created Review Draft at $REVIEW_DRAFT"
```

File: resources.whatwg.org/build/review.sh [hunk 3]
```diff
@@ -15,4 +19,7 @@ sed 's/^Group: WHATWG$/&\
...
 echo "Please verify that only one line changed relative to $INPUT_FILE:"
-diff -up "$INPUT_FILE" "$REVIEW_DRAFT" || exit 0
+diff -up "$INPUT_FILE" "$REVIEW_DRAFT"
```
Comment (domenic) [inline resources.whatwg.org/build/review.sh]:
```suggestion
echo ""
diff -up "$INPUT_FILE" "$REVIEW_DRAFT" || true
echo ""
```

**Comment by annevk** [comment]
Thanks, addressed! I also have the changes for whatwg/html locally (and tested). I guess if this is all okay I'm going to do a new PR for that one (and the other drafts).

**Pull request merged**